src/main/java/uk/gov/cshr/domain/EmailUpdate.java

```diff
@@ -20,5 +20,15 @@
 
     @ManyToOne
     private Identity identity;
+
     private Instant timestamp;
-}
\ No newline at end of file
+
+    public String toString() {
```

Comment:
Lombok Data annotation provides a toString method, is there a reason we need to provide our own?

Reply:
Same as above.